smoderp2d/providers/base/data_preparation.py

```diff
@@ -530,9 +530,10 @@ def run(self):
         dem_aspect_aoi = self._clip_raster_layer(
             dem_aspect, aoi_mask, 'dem_aspect_aoi'
         )
-        points_aoi = self._clip_record_points(
-            self._input_params['points'], aoi_polygon, 'points_aoi'
-        )
+        if self._input_params['points']:
```

Comment:
Same question as above

Reply:
Also, what about moving the entire if-clause into the next commented one? The variable is not used anywhere between, so it is probably unnecessary to execute the clause twice.

Reply:
This could be probably done. But it requires #273 to be merged before.

Reply:
OK.

Reply:
Solved in e2f294d.